retirement_api/tests/test_commands.py

```diff
@@ -24,5 +24,5 @@ def test_check_ssa_values(self, mock_run_tests):
     @mock.patch('retirement_api.management.commands.check_ssa.check_api.run')
     def test_check_ssa(self, mock_run):
         mock_run.return_value = collector
-        test_run = call_command('check_ssa')
-        self.assertTrue(mock_run.call_count == 1)
+        call_command('check_ssa')
```

Comment:
For testing management command once nice pattern we've used before is to leave in use of `self.stdout` [in the command](https://github.com/cfpb/retirement/pull/230/files#diff-a545f559cd701b4e4b60407bb10c6480L22) (thus making it work the default way writing to stdout) but then redirecting the output during tests.

[Here's](https://github.com/cfpb/cfgov-refresh/blob/62406a84a76738cdfee7bb446657f732f1660df9/cfgov/data_research/tests/management/commands/test_conference_notify.py#L32) an example test where we pass in an alternate buffer for stdout and then check its output. Another way is to set `verbosity = 0` [like this](https://github.com/cfpb/cfgov-refresh/blob/62406a84a76738cdfee7bb446657f732f1660df9/cfgov/data_research/tests/management/commands/test_conference_notify.py#L21) to disable output from the test, assuming the command supports it [somehow](https://github.com/cfpb/cfgov-refresh/blob/62406a84a76738cdfee7bb446657f732f1660df9/cfgov/data_research/management/commands/conference_notify.py#L45).

I like this pattern because it seems reasonable for the default management command behavior to write to stdout regardless of how you happen to have your logging set up.